jdi-light-mobile-examples/src/main/java/selenium/site/sections/LoginForm.java

```diff
@@ -0,0 +1,12 @@
+package selenium.site.sections;
+
+import com.epam.jdi.light.elements.composite.Form;
+import org.openqa.selenium.WebElement;
+import selenium.entities.User;
+
+public class LoginForm extends Form<User> {
+    WebElement name;
+    WebElement password;
+
+    WebElement loginButton;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5104364)